sklearn/utils/estimator_checks.py

```diff
@@ -640,7 +641,7 @@ def check_fit1d_1feature(name, estimator_orig):
 
 @ignore_warnings
 def check_fit1d_1sample(name, estimator_orig):
-    # check fitting 1d array with 1 feature
+    # check fitting 1d array with 1 sample
```

Comment:
What does this currently test? It should test that a value error is raised. That's weird. And I guess for a 1d example, it's not specified whether its a sample or a feature and therefore we error now. The test was written before we deprecated this behavior